lib/Plerd.pm

```diff
@@ -569,6 +650,70 @@ sub generates_post_guids {
          . "anyway.)";
 }
 
+# Tag-related builders & methods
+sub _build_tags_index_uri {
+    my $self = shift;
+    return $self->tag_uri("");
+}
+
+sub _build_tags_publication_path { 'tags' }
+
+sub _build_tags_publication_directory {
+    my $self = shift;
+
+    return $self->_build_subdirectory( 'tags_publication_path', 'docroot' );
+}
+
+sub _build_tags_template_file {
+    my $self = shift;
+
+    return Path::Class::File->new(
+        $self->template_directory,
+        'tags.tt',
+    );
+}
+
+# Return either the tags/index.html file
+# or a tags/TAGNAME.html file if given a tag
+sub tags_publication_file {
```

Comment:
All these public methods ought to be documented in the appropriate section of the POD at the end of this file.